test/bravo/BravoPostProposalCheck.sol

```diff
@@ -0,0 +1,23 @@
+pragma solidity ^0.8.0;
+
+import "@forge-std/Test.sol";
+
+import {Addresses} from "@forge-proposal-simulator/addresses/Addresses.sol";
+
+import {MockBravoProposal} from "proposals/MockBravoProposal.sol";
+
+// @notice this is a helper contract to execute proposals before running integration tests.
+// @dev should be inherited by integration test contracts.
+contract BravoPostProposalCheck is Test {
```

Comment:
I think we should always get the latest proposal here, instead of hardcoded the proposal. What do you think @ElliotFriedman ?